Fuse.pretty/Fuse_SunFuse-6HP.kicad_mod

```diff
@@ -0,0 +1,48 @@
+(module Fuse:Fuse_SunFuse-6HP (layer F.Cu) (tedit 5A64049B)
```

Comment:
Did you export the footprint from the footprint browser and copy the footprint by hand into the .pretty repo?

Please remove "Fuse:"